### DISCUSSION TITLE
Early stopping

### DISCUSSION THREAD

**Discussion by boutsitron** [neuraloperator/neuraloperator#339] [Q&A]
Hi guys, I would like to ask whether you have implemented early stopping in the model to be able to stop the training based on some `patience` parameter. Thanks!

**Comment by JeanKossaifi** [comment]
We don't have it directly in the library but that should be straightforward to add with just a [Callback](https://neuraloperator.github.io/neuraloperator/dev/modules/generated/neuralop.training.callbacks.Callback.html#neuralop.training.callbacks.Callback) in the trainer that keeps track of the N last losses and stops when you match the criteria.
